packages/integrations/vercel/README.md

```diff
@@ -261,6 +261,26 @@ export default defineConfig({
 });
 ```
 
+### maxDuration
+
+**Type:** `number`<br>
+**Available for:** Serverless
+
+Use this property to extend or limit the maximum duration (in seconds) that the Serverless Function can run before timing out. See [Vercel documentation](https://vercel.com/docs/functions/serverless-functions/runtimes#maxduration) to find out the valid range for your plan. 
+
+```js
```

Comment:
``````suggestion
```diff lang="js"
``````
I think we can do this, per Hippo's docs, so that it will render as nicely as possible in Docs: 
https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-text-markers/README.md#combining-syntax-highlighting-with-diff-like-syntax